app/controllers/publishers_controller.rb

```diff
@@ -94,7 +94,17 @@ def create_done
 
   def resend_auth_email    
     @publisher = Publisher.find(params[:publisher_id])
-    
+
+    @should_throttle = should_throttle_create_auth_token?
```

Comment:
Not a blocker - but let's consider adding ` || params[:captcha]` for consistency and easier testing.